### ISSUE TITLE
Dolphin/Jade img new SciLexer.dll version 4.1.5 error

### ISSUE THREAD

**Issue by PeterMoe** [jgfoster/Jade.bak#210]
Hi James,

i used the actual Dolphin Repository with Jade Repository.

After loading the Jade source paks and restarting the JadeDev.img7 immediately i got this Error:

```
24.04.2019 09:59:18: 'ScintillaView should not implement #command:id:'
ScintillaView(Object)>>error:
ScintillaView(Object)>>shouldNotImplement
ScintillaView>>command:id:
ReferenceView(View)>>wmCommand:wParam:lParam:
ReferenceView(View)>>dispatchMessage:wParam:lParam:
[] in InputState>>wndProc:message:wParam:lParam:cookie:
BlockClosure>>ifCurtailed:
GuiInputState(InputState)>>wndProc:message:wParam:lParam:cookie:
ScintillaView(ControlView)>>defaultWindowProcessing:wParam:lParam:
ScintillaView(View)>>dispatchMessage:wParam:lParam:
[] in InputState>>wndProc:message:wParam:lParam:cookie:
BlockClosure>>ifCurtailed:
GuiInputState(InputState)>>wndProc:message:wParam:lParam:cookie:
WalkbackDialog(DialogView)>>basicCreateAt:extent:
WalkbackDialog(View)>>createAt:extent:
[] in WalkbackDialog(ShellView)>>createAt:extent:
[] in ShellView class>>createHookBlock
WalkbackDialog(ShellView)>>createAt:extent:
WalkbackDialog(View)>>create
WalkbackDialog(View)>>handle
WalkbackDialog(DialogView)>>showModalTo:
WalkbackDialog(DialogView)>>showModal
Process>>showWalkback:callStack:debuggable:resumable:
Process>>walkback:topFrame:resumable:
DevelopmentSessionManager>>unhandledException:
DevelopmentSessionManager(SessionManager)>>onUnhandledError:
Error>>defaultAction
Error(Exception)>>_propagateFrom:
Error(Exception)>>_propagate
Error(Exception)>>signal
Error(Exception)>>signal:
Error class(Exception class)>>signal:
ScintillaView(Object)>>error:
ScintillaView(Object)>>shouldNotImplement
ScintillaView>>command:id:
ReferenceView(View)>>wmCommand:wParam:lParam:
ReferenceView(View)>>dispatchMessage:wParam:lParam:
[] in InputState>>wndProc:message:wParam:lParam:cookie:
BlockClosure>>ifCurtailed:
GuiInputState(InputState)>>wndProc:message:wParam:lParam:cookie:
ScintillaView(ControlView)>>defaultWindowProcessing:wParam:lParam:
ScintillaView(View)>>dispatchMessage:wParam:lParam:
[] in InputState>>wndProc:message:wParam:lParam:cookie:
BlockClosure>>ifCurtailed:
GuiInputState(InputState)>>wndProc:message:wParam:lParam:cookie:
ScintillaView(View)>>setFocus
ScintillaView(MultilineTextEdit)>>tabFocus
ShellView(ContainerView)>>tabFirst
ShellView>>setInitialFocus
SmalltalkSystemShell(Shell)>>setInitialFocus
```
Can't use actual Dolphin Repos (24 april 2019) with Jade Repos.

With the Dolphin Repos from 12 april 2019 it works fine.

Regards Peter

**Comment by brunobuzzi** [comment]
How do you get this error ?
With the lastest PR (https://github.com/jgfoster/Jade/pull/212) and Dophin fix (https://github.com/dolphinsmalltalk/Dolphin/pull/746/commits/a8ca76f7b5ff6c579a1c70974bcace7476e75170) Jade load without problems.
I just made the Jade development build

**Comment by brunobuzzi** [comment]
Ok, is the Jade.exe that produce this error but running Jade from Dolphin works ok.

**Comment by brunobuzzi** [comment]
Just copy the Dolphin SciLexer.dll (https://github.com/dolphinsmalltalk/Dolphin/blob/master/SciLexer.dll) to Jade folder ../runtime/bin and Jade.exe will login as expected.

**Comment by PeterMoe** [comment]
hi brunobuzzi,

thanx for your feedback. i think your changes will work. :)

regards Peter

**Comment by jgfoster** [comment]
I think this was fixed with 67ac915c38a9a030cc70cfadb1795d7973acfba7.
